cmake_modules/optionDefaults.cmake

```diff
@@ -2,20 +2,12 @@ if ( NOT DESTDIR )
     set( DESTDIR "" )
 endif()
 
-if ( NOT PREFIX )
-if ( WIN32 )
-    set( prog86key "ProgramFiles(x86)" )
-    if ("$ENV{${prog86key}}" STREQUAL "")
-        set( PREFIX "$ENV{ProgramFiles}" )
-    else()
-        set( PREFIX "$ENV{${prog86key}}" )
-    endif()
-    set(ORIG_PREFIX ${PREFIX})
-    string(REGEX REPLACE "\\\\" "/" PREFIX ${PREFIX})    
-else (WIN32)
-    set( PREFIX "${DESTDIR}/opt" )
-    set( ORIG_PREFIX ${PREFIX})
-endif (WIN32)
+if(NOT PREFIX)
+    if(WIN32)
+        set(PREFIX "$ENV{ProgramFiles}")
```

Comment:
Won't this be the wrong destination when building the 32bit version of the Client Tools?